web/src/components/SelectionDialog.js

```diff
@@ -21,7 +22,19 @@ class SelectionDialog extends React.Component {
     this.setState({ open: true })
   }
 
+  getMirandaURL() {
+    var url = urljoin('https://github.com/Wattpad/highlander/tree/master/k8s/charts/services', this.props.serviceName)
+    return url
+  }
+
+  getHighlanderURL() {
+    var url = urljoin('https://github.com/Wattpad/highlander/tree/', this.props.gitref, this.props.highlanderPath)
```

Comment:
I would avoid naming this single use var and just `return urljoin(...)` directly

Reply:
My bad, it was initially configured that way and then I moved it into its own var when debugging an error as it looked like `urljoin()` was not returning a string.